src/python/pants/notes/1.25.x.rst

```diff
@@ -10,6 +10,44 @@ The ``1.25.x`` series brings two major changes to Pants:
 
 Please see https://groups.google.com/forum/#!topic/pants-devel/3nmdSeyvwU0 for more information.
 
+1.25.1rc0 (03/20/2020)
+-------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Zinc: Use an Array[PublishDiagnosticsParam] for LSP compliance (#9292)
+  `PR #9292 <https://github.com/pantsbuild/pants/pull/9292>`_
+
+* Enable zinc to log diagnostics for jvm languages (#9228)
+  `PR #9228 <https://github.com/pantsbuild/pants/pull/9228>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Report warnings and errors to reporting server for JVM targets (#9293)
+  `PR #9293 <https://github.com/pantsbuild/pants/pull/9293>`_
+
+Bugfixes
+~~~~~~~~
+
+* Fix relative path in Zinc compiler (#9261)
+  `PR #9261 <https://github.com/pantsbuild/pants/pull/9261>`_
+
+* Revert "Load the bootstrapped zinc compiler from the zinc server's classpath. (#8753)" (#9226)
+  `PR #8753 <https://github.com/pantsbuild/pants/pull/8753>`_
```

Comment:
You'll probably need to delete this line

Reply:
Why is that? I did revert it on that branch. The commit that's being reverted was already present at the time of 1.25 release, in case you're thinking that wasn't the case.

Reply:
The script which generates the changelog looks for all `#\d+` patterns and adds them as links, but IIRC the thing which actually renders this rst errors if there's more than one of them.

If it works, life is good, but if you get a weird error from something trying to parse the rst, this is probably your fix :)

Reply:
Ah I see! Maybe I need to change the 8753 to 9226 then

Reply:
Ah! Already there. OK. Will delete this line! :smile:

Reply:
That'd be good. I think the idea is to usually only have one link per entry and to have that link correspond to the actual PR number.

BTW, fixing the link is as easy as swapping 8753 with 9226 everywhere you see it in the URL. GitHub URLs are very predictable. I often use this assumption to set up the changelog with anticipated PRs, i.e. PRs I know I'll merge in the next hour before cutting the release but when I still want to get up the release notes for review before then.